content/FIDO2/FIDO2_WebAuthn_Developer_Guide/WebAuthn_Server_Registration.adoc

```diff
@@ -0,0 +1,72 @@
+
+== WebAuthn Server Registration
+
+=== Initiate Registration
+To initiate the registration ceremony the application will construct a StartRegistrationOptions instance using its builder and pass that info into the RelyingParty startRegistration method. The only require parameter is a UserIdentity describing the user for which to create a credential. The id field contains the user handle. This should be a stable, unique identifier for the user - equivalent to a username. However due to privacy considerations it is recommended to set the user handle to a random byte array rather than a username.
```

Comment:
Maybe: s/pass that info into/pass that into/ ?

Reply:
incorporated into commit eacd866